geth/params/config.go

```diff
@@ -64,6 +66,27 @@ type FirebaseConfig struct {
 	NotificationTriggerURL string
 }
 
+// ReadAuthorizationKeyFile reads and loads FCM authorization key
+func (c *FirebaseConfig) ReadAuthorizationKeyFile() ([]byte, error) {
+	if len(c.AuthorizationKeyFile) <= 0 {
+		return nil, ErrEmptyAuthorizationKeyFile
```

Comment:
Hmmmm, that sounds reasonable if we need such context that is. Will change it up.